graveler/pgiterator.go

```diff
@@ -91,14 +104,15 @@ func (ri *PGRepositoryIterator) fetch(initial bool) {
 	}
 }
 
-func (ri *PGRepositoryIterator) SeekGE(id RepositoryID) bool {
+func (ri *PGRepositoryIterator) SeekGE(id RepositoryID) {
 	ri.offset = string(id)
 	ri.shouldFetch = true
 	ri.buf = make([]*RepositoryRecord, 0)
 	ri.fetch(true) // do a new initial fetch
-
+	ri.postSeekGE = true
+	ri.postInit = false
```

Comment:
If SeekGE doesn't return any value, doesn't make more sense to move the fetch into next?

Reply:
It does, fixed.